src/client/components/mapComponents/dimentionsContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
the reason I left the style out for further implementation was that I felt its dimension has nothing to do with its see-able parent (as well as the middle components AKA Map component) but should be defined when the game world is initiated. with this fix we are only hiding the dimension from the Map component. but whoever needs to use the map should have the knowledge of the dimension (e.g. if we want it to be full screen the component user should know about screen dimension). 
I don't have enough foresight about how we want to use that, but this might be OK if the decision of the map dimension is more local than global.